### ISSUE TITLE
[question] How to customize css to change colors?

### ISSUE THREAD

**Issue by hiden33** [snwh/paper-gtk-theme#147]
Hi,
Sorry to bother you with my totally newbie questions... But i want to change the colors of wingpanel & windows title bars, just to try other colors. But as i'm suck, i don't find the file to edit...

Thank you for your greeeat job, this theme is just the only one I kept for more than 4 days lol

**Comment by snwh** [comment]
Well it's not as straight forward as that –it's actually several files.

**Comment by hiden33** [comment]
Yes I know (forget an "s" to "files") that's exactly why I'm lost...

**Comment by snwh** [comment]
Plus all different window state styles normal, backdrop/unfocused. You can try a search and replace of the main window colour which is #636e73

**Comment by sencer** [comment]
I have a similar question and would really appreciate if you can direct me:

I used to remove  titles from the maximized windows by settings has_title="false" in metacity-theme.xml. Now I upgraded to gnome 3.16, and this is not working anymore. I guess I need to change something in gtk3 css files, but I couldn't find what to do. Is it something easy to modify?

**Comment by snwh** [comment]
GNOME 3.16 (using the Shell) is all client-side decorations, metacity has no control over that.

**Comment by sencer** [comment]
Yes, I noticed that. So I am asking about gtk3 css files -is that also wrong place to look at?

**Comment by snwh** [comment]
CSS are only stylesheets. This is a core change to GNOME that you can't change by tweaking a theme.

**Comment by sencer** [comment]
Hm. Seeing the lines like

    .maximized .titlebar {

I thought I could hide the window titles from css files. Is it possible the make them smaller?

**Comment by snwh** [comment]
You mean simply the words in the title or the titlebars themselves?

**Comment by sencer** [comment]
Title bar themselves. I would like to make them as thin as possible.
